test/tests/postprocessors/print_perf_data/tests

```diff
@@ -1,7 +1,16 @@
 [Tests]
-  [./test]
+  [./csv_log]
     type = CheckFiles
     input = print_perf_data.i
     check_files = print_perf_data_out.csv
   [../]
+
+  [./use_log_data_no_print]
+    type = RunApp
+    input = use_log_data_no_print.i
+    # Make sure that the perf_log table doesn't just report
+    # the same active time each timestep
+    # This RE looks for a time on subsequent lines that are identical
+    absent_out = '(\s(\S+)\s\|\n)[^\n]*\1'
```

Comment:
As a regex-challenged person, I don't really understand why this regex works.  Can you maybe give an example in the comments of the output you are trying to make sure is absent?